application/common/mail/mfa-enabled.html.php

```diff
@@ -26,4 +26,18 @@
 ?>
 <p>Dear <?= yHtml::encode($displayName) ?>,</p>
 
-<h2>TODO: Compose contents of this email message</h2>
\ No newline at end of file
+<p>
+    Congratulations! You have enabled 2-Step Verification on your <?= yHtml::encode($idpDisplayName) ?> Identity account.
+    This is a great way to protect access your account as well keep bad guys out of our corporate systems.
+</p>
+<p>
+    The next time you log in to a site using your <?= yHtml::encode($idpDisplayName) ?> Identity account you will be prompted for
+    2-Step Verification. On that screen you’ll see a checkbox already checked to remember your computer for 30 days.
```

Comment:
I don't know if it still matters, but at some point I got in the habit of replacing all fancy apostrophes `’` with normal apostrophes `'` because the fancy apostrophes can look strange on certain computers or fonts. (Same with fancy double-quotes). Again, I don't know if that's still a problem, so this change is totally up to you.

Reply:
Done